application/src/main/java/org/openhds/mobile/provider/DatabaseAdapter.java

```diff
@@ -41,6 +50,16 @@
 	public static final String KEY_SUPERVISOR_NAME = "username";
 	public static final String KEY_SUPERVISOR_PASS = "password";
 
+	//Recent form  table used for saving recent form instances based on Location hierarchy
+	private static final String FORM_TABLE_NAME = "formsubmission";
+
+	private static final String RECENT_FORM_TABLE = "recent_forms";
+	public static final String  KEY_PATH_ID = "path_id";
+	public static final String  KEY_HIERARCHY_PATH = "hierarchyPath";
+	public static final String  KEY_RECENT_FORM_PATH = "recentFormFilePath";
```

Comment:
To make it clear changed database name Association_DB and also removed 'recent' from table columns.